MainModule/Server/Core/HTTP.lua

```diff
@@ -38,7 +38,11 @@ return function(Vargs)
 	server.HTTP = {
 		Init = Init;
 		Service = service.HttpService;
+<<<<<<< Updated upstream
 		HttpEnabled = pcall(service.HttpService.GetAsync, service.HttpService, "https://google.com");
+=======
+		HttpEnabled = pcall(service.HttpService.GetAsync, service.HttpService, "http://www.google.com/robots.txt");
```

Comment:
Wouldn't `1.1.1.1` make more sense? It's resolve speed could be theoretically faster.

Nonetheless requesting robots.txt is significantly faster.

Reply:
It's easier for us to call www.google.com/robots.txt then ``1.1.1.1`` because of roblox issues.

Reply:
thonk